src/ngx_http_lua_util.c

```diff
@@ -70,6 +70,25 @@
 #endif
 
 
+#if !(NGX_WIN32)
```

Comment:
I suggest we do feature test instead of excluding just win32 here for better portability.